distribution/docker/docker-compose.yml

```diff
@@ -16,6 +16,7 @@ services:
        - cluster.routing.allocation.disk.watermark.high=1b
        - cluster.routing.allocation.disk.watermark.flood_stage=1b
        - node.store.allow_mmap=false
+       - geoip.downloader.enabled=false
```

Comment:
Why can't we pass down the `es.geoip_v2_disabled_in_tests` system property here? I think it would be great if we could do that, but I'm not sure whether it is possible.

Reply:
We can pass it in `ES_JAVA_OPTS` in line 12 (so `- "ES_JAVA_OPTS=-Xms512m -Xmx512m -Dgeoip.downloader.enabled.default=false"`) but I don't know if that buys us anything. System property is nice if you're not sure if `ingest-geoip` module is there but in docker we always have it.

Reply:
I see, then I think this is okay. And with settings this also validated at startup, so if we ever change this setting then we know quickly to update the usage of this setting here and elsewhere in packaging infra.

Reply:
yeah, I think the same